chainer/variable.py

```diff
@@ -1500,6 +1501,8 @@ def __bool__(self):
 
 
 def _backprop_to_all(outputs, retain_grad, loss_scale):
+    OrderedDict3 = chainer.utils._collections.OrderedDict
```

Comment:
This one should be renamed to `OrderedDict`, too?

Reply:
I left this name to make it clear that there is overhead in Python 2.  In the function, both `collections.OrderedDict` and `chainer.utils._collections.OrderedDict` are used.

Reply:
I'll fix this line, too, because the point I wrote seems clear from the coding style disallows `from collections import OrderedDict`.

Reply:
Maybe a single comment on this line might be helpful?